### DISCUSSION TITLE
REQ: TEAMLOCK very difficult to use with certain teamnames

### DISCUSSION THREAD

**Discussion by meag** [ezQuake/ezquake-source#496] [Ideas]
**ezQuake version:**
Not sure

**OS/device including version:**
Presumed all

**Describe the bug**
```
[23:41]eb:E.g. if I watch SR vs LA and want to teamlock on LA
[23:41]eb:They have this white center dot in their team name
[23:41]eb:CTRL + .
[23:41]eb:And red letters
[23:41]eb:If I put the teamlock on CTRL + . ALT + l ALT + a CTRL + .
[23:41]eb:It doesn't work
[23:41]eb:But maybe it's a different white dot... and not the one from CTRL + .
[23:42]eb:Hard to tell, and hard to even try out, because I cannot use the hexcode
```

**Comment by meag** [comment]
We looked into this a bit more on discord, certain characters are impossible to enter
Discussed tab-complete for teamnames - there is playername at the moment but ideally `teamlock [tab]` would start autocomplete using the team list

**Comment by elenandtha576-coder** [comment]
Ja, das macht Sinn. Wenn manche Zeichen nicht eingegeben werden können, kann das definitiv Probleme bei Teamnamen verursachen. Tab-Vervollständigung für Teamnamen wäre eine gute Verbesserung. Wenn teamlock [Tab] aus der bestehenden Teamliste ziehen würde, ginge alles schneller und Tippfehler oder ungültige Zeichen könnten vermieden lucky7even casino https://lucky7even-de.de/ werden. Da die Autovervollständigung für Spielernamen bereits funktioniert, wäre es recht einfach, diese Logik auf Teams auszuweiten und die Benutzerfreundlichkeit deutlich zu verbessern.

**Comment by ZoeyRivera** [comment]
Wygląda na to, że problem wynika z różnicy między znakami Unicode, które wizualnie są bardzo podobne, ale mają inne kody. Ten „biały punkt” w nazwie drużyny prawdopodobnie nie jest tym samym znakiem co wpisywany przez CTRL + ., dlatego teamlock go nie rozpoznaje. W ezQuake niestety brak wsparcia dla łatwego wprowadzania heksów dodatkowo [bisons.pl](https://bisons.pl/) to utrudnia. Najprostsze rozwiązanie to skopiować dokładny znak z nazwy drużyny (jeśli masz taką możliwość) albo sprawdzić ich configi. Inaczej może to po prostu nie zadziałać.
